### PR TITLE
Update //third_party:licenses

### PR DESCRIPTION
**Pull request by schroederc**

### CODE REVIEW

File: third_party/jsr250_api/BUILD [hunk 1]
```diff
@@ -0,0 +1,3 @@
+licenses(["notice"])
```
Comment (zrlk) [inline third_party/jsr250_api/BUILD]: CDDL 1.0 is reciprocal

File: third_party/libuuid/BUILD [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
```
Comment (zrlk) [inline third_party/libuuid/BUILD]: gpl2 is restricted

File: third_party/jsr250_api/BUILD (hunk 1 above)
Comment (schroederc) [reply third_party/jsr250_api/BUILD]: ✓

File: third_party/libuuid/BUILD (hunk 2 above)
Comment (schroederc) [reply third_party/libuuid/BUILD]: ✓

**Pull request merged**